src/officeGraph.ts

```diff
@@ -64,51 +82,38 @@ export class OfficeGraph {
         });
     }
 
-    readonly FilterNotCompletedAndNagMeCategory = "$filter=(status ne 'completed') and (categories/any(a:a eq 'NagMe'))";
-    readonly FilterNagMeCategory = "$filter=(categories/any(a:a eq 'NagMe'))";
-    readonly PropertyNagLast = 'String {d0ac6527-76d0-4eac-af0b-b0155e8ad503} Name NagLast';
-    readonly PropertyNagPreferences = 'String {b07fd8b0-91cb-474d-8b9d-77f435fa4f03} Name NagPreferences';  //!!! for now just a policy string.
-    readonly ExpandNagExtensions = `$expand=singleValueExtendedProperties($filter=id eq '${this.PropertyNagLast}' or id eq '${this.PropertyNagPreferences}')`;
-    readonly NagExtensions: OutlookTask = {
-        singleValueExtendedProperties: [{
-            id: "String {b07fd8b0-91cb-474d-8b9d-77f435fa4f03} Name NagPreferences",
-            value: ''
-        }, {
-            id: "String {d0ac6527-76d0-4eac-af0b-b0155e8ad503} Name NagLast",
-            value: ''
-        }]
-    };
+
 
     async setConversations(oid: string, conversations: Partial<ConversationReference>[]) {
-        let accessToken = await app.authManager.getAccessTokenFromOid(oid);
+        let accessToken = await app.authManager.getAccessTokenFromOid(oid);  //! BUG should remove authManager dependency
```

Comment:
Did you already add an issue for this?

Reply:
Yes, this is the dependency on app issue.